app-nordigen/banks/tests/ing-pl-ingbplpw.spec.js

```diff
@@ -0,0 +1,161 @@
+        bic: 'INGBPLPW',
+        transaction_total_days: '365',
+        countries: ['PL'],
+        logo: 'https://cdn.nordigen.com/ais/ING_PL_INGBPLPW.png',
+        supported_payments: {},
+        supported_features: [
+          'access_scopes',
+          'business_accounts',
+          'card_accounts',
+          'corporate_accounts',
+          'pending_transactions',
+          'private_accounts'
+        ]
+      }
+    };
+
+    it('returns normalized account data returned to Frontend', () => {
+      expect(bank.normalizeAccount(accountRaw)).toMatchInlineSnapshot(`
+          {
+            "account_id": "d3eccc94-9536-48d3-98be-813f79199ee3",
+            "institution": {
+              "bic": "INGBPLPW",
+              "countries": [
+                "PL",
+              ],
+              "id": "ING_PL_INGBPLPW",
+              "logo": "https://cdn.nordigen.com/ais/ING_PL_INGBPLPW.png",
+              "name": "ING",
+              "supported_features": [
+                "access_scopes",
+                "business_accounts",
+                "card_accounts",
+                "corporate_accounts",
+                "pending_transactions",
+                "private_accounts",
+              ],
+              "supported_payments": {},
+              "transaction_total_days": "365",
+            },
+            "mask": "4321",
+            "name": "Current Account for Individuals (Retail) (XXX 4321)",
+            "official_name": "Current Account for Individuals (Retail)",
+            "type": "checking",
+          }
+        `);
+    });
+  });
+
+  describe('#sortTransactions', () => {
+    it('sorts transactions by time and sequence from newest to oldest', () => {
+      const transactions = [
+        { transactionId: 'D202301180000003' },
+        { transactionId: 'D202301180000004' },
+        { transactionId: 'D202301230000001' },
+        { transactionId: 'D202301180000002' },
+        { transactionId: 'D202301200000001' }
+      ];
+      const sortedTransactions = bank.sortTransactions(transactions);
+      expect(sortedTransactions).toEqual([
+        { transactionId: 'D202301230000001' },
+        { transactionId: 'D202301200000001' },
+        { transactionId: 'D202301180000004' },
+        { transactionId: 'D202301180000003' },
+        { transactionId: 'D202301180000002' }
+      ]);
+    });
+
+    it('handles empty arrays', () => {
+      const transactions = [];
+      const sortedTransactions = bank.sortTransactions(transactions);
+      expect(sortedTransactions).toEqual([]);
+    });
+
+    it('returns empty array for undefined input', () => {
+      const sortedTransactions = bank.sortTransactions(undefined);
+      expect(sortedTransactions).toEqual([]);
+    });
+  });
+
+  describe('#countStartingBalance', () => {
+    it('should calculate the starting balance correctly', () => {
+      const sortedTransactions = [
+        {
+          transactionAmount: { amount: '-100.00', currency: 'USD' },
```

Comment:
Do any of these banks do multiple currencies in the same account? How would that be handled?

Reply:
I have no idea. The Actual supports for now only single currency so I totally skipped this part.